package.json

```diff
@@ -365,7 +365,8 @@
       "typings"
     ],
     "nohoist": [
-      "@fluentui/web-components/@storybook/html"
+      "@fluentui/web-components/@storybook/html",
+      "@fluentui/web-components/@typescript/lib-dom"
```

Comment:
The `@typescript/lib-dom` package is included in `nohoist`, since `@types/web` breaks in packages which target ES5. This ensures that this types override is scoped only to the web components package.

https://github.com/microsoft/TypeScript-DOM-Lib-Generator?tab=readme-ov-file#typeslib-minimum-target
![image](https://github.com/user-attachments/assets/dde75ec3-98a8-4c7d-813e-129d5e591ad7)

Reply:
Thanks for this - makes sense!